tablespace/benchmark.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+wget --no-verbose --continue 'https://datasets.clickhouse.com/hits_compatible/hits.tsv.gz'
+guzip hits.tsv.gz
+chmod 666 hits.tsv
+
+psql 'host=<hostname> port=5432 dbname=csdb user=csuser password=<password> sslmode=require' < create.sql
```

Comment:
It will be nice to use env variables rather than asking to replace <hostname> and <password> from the script.

Reply:
Just having well defined variables in bash script should be enough. Please see updated and cleaned **benchmark.sh** files. This pattern follows other benchmark scripts.